hf_xet/src/lib.rs

```diff
@@ -41,15 +76,15 @@ pub fn upload_files(
     token_refresher: Option<Py<PyAny>>,
     progress_updater: Option<Py<PyAny>>,
     _repo_type: Option<String>,
-) -> PyResult<Vec<PyPointerFile>> {
+) -> PyResult<Vec<PyXetUploadInfo>> {
```

Comment:
nit: `PyXetUploadInfo` doesn't have the same fields as `PyPointerFile` so this is technically breaking the interface as someone could have depended on the `filesize` or `path` field (or that the returned type is a `PyPointerFile`)...

Reply:
I understand. I'm not too concerned about this change. None of our python code is using the return values at the moment. 

While there could be a client that's using hf-xet directly, I'm very doubtful that they are using the return values either.